.travis.yml

```diff
@@ -36,6 +36,7 @@ after_success:
 
 before_deploy:
   - make build_redist
+  - for f in faas-cli*; do shasum -a 256 ./$f | awk '{print $1}' > ./$f.sha256; done
```

Comment:
LGTM but I would rather this was in a shell script for portability. If we don't have a ci or hack folder, please create one for it.

Alex

Reply:
OK